content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -92,6 +92,18 @@ Common forms of values include:
 
 One of the [recommended labels](/docs/concepts/overview/working-with-objects/common-labels/#labels).
 
+### kubectl.kubernetes.io/restartedAt
+
+Example: `kubectl.kubernetes.io/restartedAt: "2023-03-31T15:30:00Z"`
+
+Used on: Pod
+
+In this example, the annotation indicates that the Pod was most recently restarted at 3:30 PM UTC on March 31st, 2023.
+
+This annotation is used by kubernetes to keep track of the timestamp of the most recent container restart for a particular pod. This information can be useful for debugging and monitoring purposes.
```

Comment:
Following the trend set by some of the other entries, this section could include which commands would trigger the addition of this annotation.